services/algolia-sync/src/components/graphql/query.js

```diff
@@ -0,0 +1,21 @@
+const createClient = require('./create-client');
+const { GRAPHQL_URI } = require('../../env');
+
+const queryFromBase = async (query, tenantKey, input) => {
+  const context = {
+    headers: {
+      'x-tenant-key': tenantKey,
+    },
+  };
+
+  try {
```

Comment:
This try/catch block is redundant. Remove :)

Reply:
Done